src/EditorFeatures/VisualBasicTest/LineCommit/CommitOnEnterTests.vb

```diff
@@ -76,7 +76,8 @@ End Class
                            </Project>
                        </Workspace>
 
-            AssertCommitsStatement(test, expectCommit:=False)
+            AssertCommitsStatement(test, _ ' Test that Roslyn supports VB16 Continuation Comments After Line Continuation internally
+                                   expectCommit:=False)
```

Comment:
can you undo this, it seems unnecessary.

Reply:
(and i don't think is a style we would allow in teh codebase).

Reply:
It's not, It is just a test that proves VB16 works. The change to VB16 is needed for #5056. Do you Just not want any tests?

Reply:
> Do you just not want any tests?

This isn't really a test.  You're editing actual code here.  I don't like thsi style and would strongly push for it to not be allowed in the IDE codebase.  Thanks :)

Reply:
Deleted will allow anything that depends on VB16 to be the place that complains if VB LangVersion ever set <16